<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineSelectionDialog</class>
 <widget class="QDialog" name="LineSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>547</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>609</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selección de línea</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="linesGroupBox">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>LINE</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="redRadioButton">
        <property name="text">
         <string>RED</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="blueRadioButton">
        <property name="text">
         <string>BLUE</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="greenRadioButton">
        <property name="text">
         <string>GREEN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cyanRadioButton">
        <property name="text">
         <string>CYAN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="magentaRadioButton">
        <property name="text">
         <string>MAGENTA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="yellowRadioButton">
        <property name="text">
         <string>YELLOW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4" colspan="2">
    <widget class="QLabel" name="image">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>326</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>LINE_CANCEL</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>416</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>LINE_OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="completeUp">
     <property name="text">
      <string>GO_UP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="completeDown">
     <property name="text">
      <string>GO_DOWN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
